torch/_inductor/triton_ops/autotune.py

```diff
@@ -204,6 +204,50 @@ def run(self, *args, grid, stream):
         return result
 
 
+def _find_names(obj):
+    import gc
+    import inspect
+
+    frame = inspect.currentframe()
+    for frame in iter(lambda: frame.f_back, None):
+        frame.f_locals
+    obj_names = []
+    for referrer in gc.get_referrers(obj):
+        if isinstance(referrer, dict):
+            for k, v in referrer.items():
+                if v is obj:
+                    obj_names.append(k)
```

Comment:
Well, we could pass it down in the codegen to the CachedAutotuner object, but this was a less intrusive way of doing the same thing.

Since it's debug/instrumentation code, I didn't feel like it was necessarily worth doing the modification.